packages/api-fetch/src/middlewares/preloading.js

```diff
@@ -32,15 +27,12 @@ const createPreloadingMiddleware = ( preloadedData ) => ( options, next ) => {
 			.join( '&' );
 	}
 
-	if ( typeof options.path === 'string' ) {
+	if ( typeof options.path === 'string' && options.parse !== false ) {
```

Comment:
The idea is that it's clearer this way that it's opt-out and not opt-in. The default value allows us to change the condition but I think it's a little bit safer to check against false values on "opt-out" flags.

Reply:
Right, you need to pass `parse: false` to opt-out, that makes sense. But why not make the condition check `if ( typeof options.path === 'string' && options.parse ) {`?

I just find it easier to read that we're checking for true rather than for "not false".

Reply:
👍 updating

Reply:
Noting that if `options.parse` is undefined `options.parse` returns false but in the current condition it returns `true` so we have to assign a default value before simplifying the check.

Reply:
Oh yeah, sorry, I should have said that; it was implied but I should've mentioned it explicitly. My bad 😞

I think that'll make the code easier to follow and the intent clearer as well. 👍